src/components/Dashboard/FilingHistoryList/filings/LimitedRestoration.vue

```diff
@@ -36,7 +36,7 @@ export default class LimitedRestoration extends Vue {
 
   /** The legal name from the restoration filing. */
   get legalName (): string {
-    return this.filing.data?.restoration?.legalName || '[unknown]'
+    return this.filing.data?.restoration?.toLegalName || '[unknown]'
```

Comment:
Now displays this:

![image](https://user-images.githubusercontent.com/10002889/229894508-d553b10c-7214-402c-a54a-b509ccc84465.png)